### ISSUE TITLE
[Bug]: Terminal is not transparent in VSCode 1.92

### ISSUE THREAD

**Issue by yutayouguan** [illixion/vscode-vibrancy-continued#155]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

<img width="1037" alt="image" src="https://github.com/user-attachments/assets/f7af2e72-ed7a-4ee2-b7ec-26e8d4e2c23c">

### Environment

```markdown
- OS: Darwin arm64 23.5.0
- VSCode: 1.92.0 (Universal)
- Extension: 1.1.35
- Theme: Dark++
```

### Anything else?

<img width="1037" alt="image" src="https://github.com/user-attachments/assets/fcd0a075-0a41-43bc-af6a-09049496628b">

**Comment by illixion** [comment]
I've encountered this as well, @A2L265 [has recently made a PR](https://github.com/illixion/vscode-vibrancy-continued/pull/154) with a configuration change that fixes this:

> open the settings file and add `"terminal.background": "#00000000"` inside `"workbench.colorCustomizations"`.
> ```json
> "workbench.colorCustomizations": {
>     "terminal.background": "#00000000"
>   }
> ```

I'll look into automating this config change.

**Comment by mkanet** [comment]
I can also confirm this.  I used to be able to use:  `"terminal.integrated.rendererType": "dom"`.  However, that is not allowed anymore. 

@illixion Please let us know when you have a fix for this.  I would be very grateful!

**Comment by yutayouguan** [comment]
Thanks, the way can it work as well, and as the below setup can apply all profiles. 
``` json
  "workbench.colorCustomizations": {
    "terminal.background": "#00000000",
  },
  "workbench.settings.applyToAllProfiles": [
    "workbench.colorCustomizations",
  ]
```

**Comment by illixion** [comment]
Just published v1.1.36 which automates these config changes, feel free to reopen if this issue re-appears.
